scss/_forms.scss

```diff
@@ -123,12 +123,12 @@ select.form-control {
 }
 
 
-// Static form control text
+// Readonly controls as plain text
 //
-// Apply class to an element to make any string of text align with labels in a
-// horizontal form layout.
+// Apply class to a readonly input to make it appear like regular plain
+// text (without any border, background color, focus indicator)
 
-.form-control-static {
+.form-control-readonly-plain {
```

Comment:
Would it be enough to call this `.form-control-readonly` instead? Do we envision having a readonly and readonly-plain in the future? Or would `.form-control-plaintext` be better?